tests/nightly/model_backwards_compatibility_check/common.py

```diff
@@ -165,3 +177,163 @@ def forward(self, x):
         x = F.tanh(self.fc1(x))
         x = F.tanh(self.fc2(x))
         return x
+
+class Dictionary(object):
+    def __init__(self):
+        self.word2idx = {}
+        self.idx2word = []
+
+    def add_word(self, word):
+        if word not in self.word2idx:
+            self.idx2word.append(word)
+            self.word2idx[word] = len(self.idx2word) - 1
+        return self.word2idx[word]
+
+    def __len__(self):
+        return len(self.idx2word)
+
+class Corpus(object):
+    def __init__(self, path):
+        self.dictionary = Dictionary()
+        self.download_data_from_s3()
+        self.train = self.tokenize(path + 'train.txt')
+        self.valid = self.tokenize(path + 'valid.txt')
+        self.test = self.tokenize(path + 'test.txt')
+
+    def download_data_from_s3(self, ):
+        print ('Downloading files from bucket : ptb-small-dataset' )
+        bucket = s3.Bucket('ptb-small-dataset')
```

Comment:
where are we downloading this from? since we can't host the ptb dataset ourselves because of the license, I wonder if it is a good idea to depend on it.

Reply:
Oh. I wasn't aware of the Penn Tree Bank dataset license issue. I had uploaded a trimmed down version of PTB to an S3 bucket. The trimmed down version was 10k training sentences and 3k each for test and validation. 
Can you suggest a different dataset which we can host ?

Reply:
See the dataset introduced in #11435

Reply:
Thanks. Replaced PTB with Sherlock in : c3c91295b7c4e11241b5a32d4885c3d775f052b6